docker-compose.yml

```diff
@@ -1,15 +1,80 @@
 version: "3.1"
 
 services:
+  conseil:
+    image: cryptonomictech/conseil:0.2002.0020-SNAPSHOT
+    ports:
+      - 80:80
+    environment:
+#  List of supported environment variables:
+#  DBHOST=${DB_Host:=db}
+#  DBUSER=${DB_User:=user}
+#  DBPW=${DB_Password:=password}
+#  DBDB=${DB_Database:=conseil}
+#  DBPORT=${DB_Port:=5432}
+#  XTZSCHEME=${XTZ_Scheme:=http}
+#  XTZHOST=${XTZ_Host:=node}
+#  XTZPREFIX=${XTZ_Prefix:=}
+#  XTZPORT=${XTZ_Port:=8732}
+#  XTZNET=${XTZ_Network:=mainnet}
+#  APIPORT=${API_PORT:=80}
+#  APIKEY=${API_KEY:=conseil}
+      DB_Host: conseil-postgres
+      DB_User: conseiluser
+      DB_Password: p@ssw0rd
+      DB_Database: conseil-local
+      DB_Port: 5432
+      XTZ_Scheme: https
+      XTZ_Host: tezos-staging.cryptonomic-infra.tech
+      XTZ_Port: 443
+      XTZ_Network: mainnet
+    entrypoint: ["/root/wait-for.sh", "conseil-postgres:5432", "-t", "120", "--", "/root/entrypoint.sh", "conseil"]
```

Comment:
Would it be possible to make the timeout setting i.e `-t` configurable somehow?

Reply:
do you mean like argument to the `docker-compose`? in this line there is ` "-t", "120",` which means timeout is 120 seconds

Reply:
Now that i think about it, perhaps that would be overkill. Someone can hand modify this anyways.